app/src/main/kotlin/com/wire/android/ui/theme/Theme.kt

```diff
@@ -15,10 +15,18 @@ fun WireTheme(
     useDarkColors: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    CompositionLocalProvider(LocalWireColors provides if (useDarkColors) DarkWireColors else LightWireColors) {
+    val wireColorScheme = WireColorSchemeTypes.current
+    val wireTypography = WireTypographyTypes.current
+    val wireDimensions = WireDimensionsTypes.current.current
```

Comment:
why current current here ?

Reply:
it's nested, so actually it is `ScreenSizeDependent<OrientationDependent<WireDimensions>>` so we have to get the current "screen size dependent" item and then inner current "orientation dependent" item

Reply:
Shouldn't we rename it then to `currentScreen` and `currentOrientation` to avoid confusion?

Reply:
Good idea - renamed them